### PR TITLE
{devel,math}[GCCcore/13.2.0] Doxygen v1.9.8, GDRCopy v2.4, Eigen v3.4.0, GMP v6.3.0, libiconv v1.17, MPFR v4.2.1, PCRE2 v10.42, SWIG v4.1.1

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in total)
alvis-c1 - Linux Rocky Linux 8.8, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/Micket/05d42e76bf249b01a9a32477f0f6efb8 for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/m/MPFR/MPFR-4.2.1-GCCcore-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+version = '4.2.1'
+
+homepage = 'https://www.mpfr.org'
+
+description = """
+ The MPFR library is a C library for multiple-precision floating-point
+ computations with correct rounding.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/m/MPFR/MPFR-4.2.1-GCCcore-12.3.0.eb:13]:
```suggestion
toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
```

**Review by SebastianAchilles — DISMISSED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
node3100.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/643225ca0b22d3a64ad6f44e57455923 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
zen2-ubuntu-2310 - Linux Ubuntu 23.10 (Mantic Minotaur), x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.11.6
See https://gist.github.com/SebastianAchilles/c515bf5248dd6b6c32fb0f28ecc4cfea for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Micket!

**Pull request merged**